pom.xml

```diff
@@ -464,6 +471,23 @@
         </plugins>
       </build>
     </profile>
+    <profile>
+      <!-- To activate use -DproductizedOpenshift (not -P) -->
```

Comment:
Non-blocker, but why did you set it to activate on a property?

Reply:
because -DproductizedOpenshift actives profile openshift too (which it needs because the cargo tests will fail otherwise)